src/renderer/pages/Home.js

```diff
@@ -40,18 +40,18 @@ function renderLandingPage(navigateTo) {
             steps={TypicalSteps}
             loop={1}
             wrapper="span"
-            className="font-semibold text-4xl mx-auto text-center text-gray-200"
+            className="block font-semibold text-4xl mx-auto text-center text-gray-200"
           />
         </Reveal>
         <Reveal delay={300}>
           <h2 className="text-gray-200 text-lg mt-2 text-center">
             Setup SSH keys in seconds 🚀
           </h2>
         </Reveal>
-        <div className="mt-12 flex flex-row justify-between items-center ">
+        <div className="mt-12 grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-5">
           {steps.map((step, index) => (
             <Reveal delay={400 + (index + 1) * 100}>
-              <div className="w-48 h-48 mr-8 flex flex-col items-center border-blue-500 border-r-2 border-b-2 rounded-lg shadow-2xl bg-gray-900">
+              <div className="w-48 h-48 flex flex-col items-center border-blue-500 border-r-2 border-b-2 rounded-lg shadow-2xl bg-gray-900">
```

Comment:
Love 🤩 how using Css Grid's grid gap property we could get rid of this one annoyance we've with flex box of defining right or left margins on individual components.

Reply:
Yup!